samples/core/loop_output/loop_output.py

```diff
@@ -12,22 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from kfp.deprecated import components
-from kfp.deprecated import dsl
+import os
+from kfp import dsl
 
+# In tests, we install a KFP package from the PR under test. Users should not
+# normally need to specify `kfp_package_path` in their component definitions.
+_KFP_PACKAGE_PATH = os.getenv('KFP_PACKAGE_PATH')
 
-@components.create_component_from_func
+
+@dsl.component(kfp_package_path=_KFP_PACKAGE_PATH)
```

Comment:
Consider omitting use of `kfp_package_path` since this is a sample not a test.

Reply:
Thanks for the comment, just addressed.